ballerina/Package.md

```diff
@@ -0,0 +1,71 @@
+## Package Overview
```

Comment:
IINM we dont need to duplicate `Module.md` content under the `Package.md` anymore, as the `Package.md` content are not displayed in the central with the recent changes. 
@anuruddhal what would be the new recommended structure for the `Module.md`, `Package.md` and `README.md` files?

Reply:
We still need to duplicate. Lang level support is not yet available.

Reply:
@anuruddhal Ack